<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="bgwidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1200</width>
      <height>800</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#bgwidget{
background-color: qlineargradient(spread:pad, x1:0.045, y1:0.0397727, x2:1, y2:1, stop:0 rgba(227, 227, 227, 255), stop:1 rgba(93, 104, 116, 255));}</string>
    </property>
    <widget class="QPushButton" name="createButton">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>580</y>
       <width>271</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#createButton {
    border-radius: 20px;
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(250, 217, 97, 255), stop:1 rgba(247, 107, 28, 255));
    font: 75 17pt &quot;Berlin Sans FB&quot;;
}

QPushButton#createButton:hover {

	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 	 		rgba(176, 149, 53, 255), stop:1 rgba(173, 74, 0, 255));
}
</string>
     </property>
     <property name="text">
      <string>Create</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordTxtInput">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>340</y>
       <width>271</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="usernameTxtInput">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>340</y>
       <width>271</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>230</y>
       <width>331</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 20pt &quot;Berlin Sans FB&quot;; color: rgb(0, 0, 0)</string>
     </property>
     <property name="text">
      <string>Create your account</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>310</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Berlin Sans FB</family>
       <pointsize>15</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 15pt &quot;Berlin Sans FB&quot;; color: rgb(0, 0, 0)</string>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>310</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 15pt &quot;Berlin Sans FB&quot;; color: rgb(0, 0, 0)</string>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="backButton">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>670</y>
       <width>271</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#backButton{
	border-radius: 20px;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 			rgba(250, 217, 97, 255), stop:1 rgba(247, 107, 28, 255));
	font: 75 17pt &quot;Berlin Sans FB&quot;;
} 

QPushButton#backButton:hover {

	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(176, 149, 53, 255), stop:1 rgba(173, 74, 0, 255));
}</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QLabel" name="errorLabel">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>540</y>
       <width>581</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;; color:red;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="bgwidget_2" native="true">
     <property name="geometry">
      <rect>
       <x>1180</x>
       <y>90</y>
       <width>1200</width>
       <height>800</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#bgwidget{
background-color: qlineargradient(spread:pad, x1:0.045, y1:0.0397727, x2:1, y2:1, stop:0 rgba(227, 227, 227, 255), stop:1 rgba(93, 104, 116, 255));}</string>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>10</y>
        <width>162</width>
        <height>66</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 45pt &quot;Berlin Sans FB&quot;;
</string>
      </property>
      <property name="text">
       <string>Sign in</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="signInButton_2">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>580</y>
        <width>271</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#signInButton {
    border-radius: 20px;
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(250, 217, 97, 255), stop:1 rgba(247, 107, 28, 255));
    font: 75 17pt &quot;Berlin Sans FB&quot;;
}

QPushButton#signInButton:hover {

	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 	 		rgba(176, 149, 53, 255), stop:1 rgba(173, 74, 0, 255));
}
</string>
      </property>
      <property name="text">
       <string>Sign In</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>90</y>
        <width>131</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>Sprites/icon_app/icon_app_27.png</pixmap>
      </property>
     </widget>
     <widget class="QLineEdit" name="passwordTxtInput_2">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>450</y>
        <width>271</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="usernameTxtInput_2">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>340</y>
        <width>271</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>240</y>
        <width>331</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 20pt &quot;Berlin Sans FB&quot;; color: rgb(0, 0, 0)</string>
      </property>
      <property name="text">
       <string>Sign in to an existing account</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>740</y>
        <width>331</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Berlin Sans FB</family>
        <pointsize>18</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 18pt &quot;Berlin Sans FB&quot;; color: rgb(0, 0, 0)</string>
      </property>
      <property name="text">
       <string>Forgot Username / Password?</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>310</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Berlin Sans FB</family>
        <pointsize>15</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 15pt &quot;Berlin Sans FB&quot;; color: rgb(0, 0, 0)</string>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>420</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 15pt &quot;Berlin Sans FB&quot;; color: rgb(0, 0, 0)</string>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="backButton_2">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>670</y>
        <width>271</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#backButton{
	border-radius: 20px;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 			rgba(250, 217, 97, 255), stop:1 rgba(247, 107, 28, 255));
	font: 75 17pt &quot;Berlin Sans FB&quot;;
} 

QPushButton#backButton:hover {

	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(176, 149, 53, 255), stop:1 rgba(173, 74, 0, 255));
}</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="errorLabel_2">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>530</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;; color:red;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="recoveryPasswordTxtInput">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>460</y>
       <width>271</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>430</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 15pt &quot;Berlin Sans FB&quot;; color: rgb(0, 0, 0)</string>
     </property>
     <property name="text">
      <string>Recovery Answer</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>430</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 15pt &quot;Berlin Sans FB&quot;; color: rgb(0, 0, 0)</string>
     </property>
     <property name="text">
      <string>Select Recovery Phrase</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="recoveryPhraseComboBox">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>460</y>
       <width>271</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(255, 255, 255)</string>
     </property>
     <item>
      <property name="text">
       <string>What is your favourite sports team?</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>What is your mother's maiden name?</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Where was your first holiday location?</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>What was the name of your first pet?</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>What was your first car?</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>What city were you born in?</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>What is your favorite book?</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>What is your favorite hobby?</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>What is your favourite band?</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>When was your highscool graduation?</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>40</y>
       <width>241</width>
       <height>181</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>Sprites/icons/user (6).png</pixmap>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
